### DISCUSSION TITLE
Aimbot

### DISCUSSION THREAD

**Discussion by killer33312** [maecry/asphyxia-cs2#6] [Ideas]
hey guys,

what about adding a aimbot & triggerbot to this cheat with weapon groups maybe?
i'm not a coder but i tought something about mouse event aimbot to be undetected?

**Comment by maecry** [comment]
might consider add that yourself as this is just a base to help you develop your cheat
basic aimbot is pretty open source and everywhere + shouldn't be that hard to add it yourself

also this based hooked createmove already so why would you do it on mouseevent

**Comment by killer33312** [comment]
hey thanks for your reply,
i'm new to developing at all but do you mean making a basic aimbot by viewangles? does in not give u red trust/vac? considering doing something like a legit cheat

**Reply by maecry** [reply]
yes and redtrust/vac etc is your end at this point as i only provided the base, do whatever with it
